### ISSUE TITLE
Implement an error page

### ISSUE THREAD

**Issue by Palbolsky** [PokemonWorkshop/PokemonStudio#672]
As a User
I would like app to give me some info when it crashes
So that I know what is the problem or at least I can go to the support and report the crash

> [!TIP]
> Maybe use the 404 or 500th Pokémon to add a design to make errors easy to understand for users

## Context

When Studio crashes in production, we have only a black screen.
An error page is necessary to inform users that a problem has occurred and provide a link to Discord server support.
